divide_and_conquer/Strassen’s Matrix Multiplication.cpp

```diff
@@ -0,0 +1,182 @@
+#include <bits/stdc++.h>
+using namespace std;
+
+#define ROW_1 4
+#define COL_1 4
+
+#define ROW_2 4
+#define COL_2 4
+
```

Comment:
Would it improve readability to use something like `using Matrix = vector<vector<int>>;` and replace all occurences of  `vector<vector<int>>`  accordingly?